.binder/README.md

```diff
@@ -0,0 +1,31 @@
+# Docker Image for Binder
+
+<!--
+This is derived from the following setup:
+https://github.com/RobotLocomotion/drake/tree/dc2a9394d/.binder
+-->
+
+*Note that due to Binder conventions, this directory MUST always be in the root
+of the repository and named either `binder` or `.binder`. This image is NOT
+intended for use by most developers or users.*
+
+These instructions are for running the image locally. For Binder itself, you
+should only need to visit the link from the root-level README.
+
+To create a Docker image and run a Docker container similar to those used by
+[Binder](https://mybinder.org) for local debugging purposes, execute the
+following `build` and `run` commands from the top level of this Git repository:
+
+```bash
+docker build -f .binder/Dockerfile -t binder .
+docker run --rm -it --name mybinder -p 8888:8888 binder
+```
+
+For the URLs printed, only open the `127.0.0.1:8888` URL (including the login
```

Comment:
Any way to either make the rest of the urls work, or exclude the broken urls from the printed list?

Reply:
I don't think so / don't think it's worth spending time on.

This is just for local development; actually using binder.org will shield users from it.